### ISSUE TITLE
Please add a verbose mode

### ISSUE THREAD

**Issue by nMustaki** [Graylog2/collector-sidecar#114]
### Problem description
Hi,

first thanks you for graylog, I really love it !

I'm trying (without much success) to configure collector-sidecar, and I am missing a verbose mode to understand what is happening (or not). I would love to have:
- warnings if the log files configurated do not exist
- error when multiple outputs are configured on graylog because filebeat only handles one
- info when log sending is not working
- info when log sending is being done (a verbose++ option)
- info when log collecting is being done (a verbose++ option)
- etc

Thanks again !

### Steps to reproduce the problem

1. Try to configure multiple logs files input with various outputs to a remote server with ssl

### Environment

* Sidecar Version: 0.1.0-beta.1
* Graylog Version:  2.1
* Operating System: Ubuntu server 16.04 + docker
* Elasticsearch Version:
* MongoDB Version:

**Comment by mariussturm** [comment]
Thanks for the feedback, this is related to #83

**Comment by mariussturm** [comment]
@nMustaki in beta.3 we added a central directory with log files from the collector backends as well as the sidecar itself. In that way all requested information are at one place without the need to reimplement parts of the backends in the sidecar itself. Like checking for a working Gelf connection, with can be e.g. in the Filebeat log file. A bad connection to the Graylog API on the other hand can be debugged with the Sidecar log file. A -debug option for verbose output was also added.
